pyregion/ds9_region_parser.py

```diff
@@ -152,119 +153,54 @@ def convert_attr(self, l):
                 yield l1, c1
 
     @staticmethod
-    def sky_to_image(l, header, rot_wrt_axis=1):
-        """
+    def sky_to_image(shapelist, header):
```

Comment:
Personally I almost always call Python functions / methods by keyword argument, as I find the resulting code more readable and less brittle for libraries where I might change the order of arguments.

So changing the argument from `sky_to_image(l)`to `sky_to_image(shapelist)` could break my or other user's scripts. Please point out such (slightly) backward-incompatible changes in the changelog, or, if it's not too much work, split them out into a separate PR that is dedicated to those argument name cleanup changes and has a dedicated changelog entry in the 2.0 section.

I would prefer `shape_list` everywhere over `shapelist`, but I don't feel strongly about this, no need to argue if you prefer `shapelist`.